src/engine/basics/Point.hxx

```diff
@@ -24,6 +25,9 @@ public:
   /// The height level.
   int height;
 
+  /// The raw height.
+  double rawHeight;
```

Comment:
What exactly is the raw height? I know it's different from height, but how? That might be something you could put in this docstring.

Reply:
Actually height is a int variable converted from rawHeight, but height is not enough togenerate the rivers  through the gradient descending, as heights in an area are the same. so we need more accurate height: rawHeight.

And we should also keep height, as other parts need an integer height to populate terrain.

Reply:
Yeah, I don't think we should get rid of it, I just wanted clarification on what the difference was between the two.